tests/test_item.py

```diff
@@ -0,0 +1,187 @@
+import json
+from peewee import SqliteDatabase
+from http.client import CREATED
+from http.client import NO_CONTENT
+from http.client import NOT_FOUND
+from http.client import OK
+from http.client import BAD_REQUEST
+from models import Item
+from app import app
+import uuid
+
+
+class TestItems:
+    def setup_class(cls):
+        Item._meta.database = SqliteDatabase(':memory:')
+        Item.create_table()
+        cls.app = app.test_client()
+
+    def setup_method(self):
+        Item.delete().execute()
+
+    def test_get_items__empty(self):
+        resp = self.app.get('/items/')
+        assert resp.status_code == OK
+        assert json.loads(resp.data.decode()) == []
+
+    def test_get_items__not_empty(self):
+        obj1 = Item.create(
+            item_id=uuid.uuid4(),
+            name='cubo',
+            price=5,
+            description='dhfsdjòfgjasdògj'
+        )
+
+        obj2 = Item.create(
+            item_id=uuid.uuid4(),
+            name='cubo',
+            price=15,
+            description='desc2'
+        )
+
+        resp = self.app.get('/items/')
+        assert resp.status_code == OK
+        assert json.loads(resp.data.decode()) == [obj1.json(), obj2.json()]
+
+    def test_post_create_item__success(self):
+        source_item = {
+            'name': 'cubo',
+            'price': 15,
+            'description': 'desc1'
+        }
+
+        resp = self.app.post('/items/', data=source_item)
+        assert resp.status_code == CREATED
+        resp2 = self.app.get('/items/')
+
+        db_data = {
+            'name': json.loads(resp2.data.decode())[0]['name'],
+            'price': json.loads(resp2.data.decode())[0]['price'],
+            'description': json.loads(resp2.data.decode())[0]['description']
+        }
+
+        assert source_item == db_data
```

Comment:
In questo test, non c'è niente che mi dica che i dati sono salvati correttamente su database. In aggiunta a quanto fatto, è necessario controllare che su database siano presenti i dati inviati con la post.